src/main/java/io/xpring/xrpl/Signer.java

```diff
@@ -0,0 +1,24 @@
+package io.xpring.xrpl;
+
+import rpc.v1.TransactionOuterClass.Transaction;
+import io.xpring.xrpl.javascript.JavaScriptSigner;
+
+public class Signer {
+    /** Please do not instantiate this static utility class. */
+    private Signer() {}
+
+    /**
+     * Sign the given transaction with the given wallet.
+     * @param transaction The transaction to sign.
+     * @param wallet The wallet that will sign the transaction.
+     * @return A `SignedTransaction`.
+     */
+    public static byte[] signTransaction(Transaction transaction, Wallet wallet) {
+        try {
+            JavaScriptSigner javascriptSigner = new JavaScriptSigner();
```

Comment:
Would it make sense to create this once as a `static final` field? It seems like it takes several seconds to construct because of the GraalVM JS loading. That could cause extra slowness if multiple transactions need to be signed.

Reply:
Fixed.  thanks for the insight.